main/menu.js

```diff
@@ -17,12 +17,32 @@ var dialog = require('./dialog')
 var shell = require('./shell')
 var windows = require('./windows')
 var thumbnail = require('./thumbnail')
+var State = require('../renderer/lib/state')
 
-var menu
+var menu, state
 
 function init () {
   menu = electron.Menu.buildFromTemplate(getMenuTemplate())
   electron.Menu.setApplicationMenu(menu)
+
+  State.load(onState)
```

Comment:
We should be able to select VLC as default player. This would be especially good in combination with autoplay for stream links (issue #671).